content/en/localstack/configuration.md

```diff
@@ -181,7 +181,7 @@ While the ElasticSearch API is actively maintained, the configuration variables
 
 | Variable | Example Values | Description |
 | - | - | - |
-| `S3_DIR` | | Configure a global parent directory that contains all buckets as sub-directories (`S3_DIR=/path/to/root`) or an individual directory that will get mounted as special bucket names (`S3_DIR=/path/to/root/bucket1:bucket1`) |
+| `S3_DIR` | | Configure a global parent directory that contains all buckets as sub-directories (`S3_DIR=/path/to/root`) or an individual directory that will get mounted as special bucket names (`S3_DIR=/path/to/root/bucket1:bucket1`). Only available for Localstack Pro. |
```

Comment:
Hi Localstack Team. Please clarify if this feature is for Pro only or not. I was having trouble getting this working in the free tier and I think the original PR adding this documentation mentions that this is a Pro-only feature. Admittedly, I was trying multiple things when I added my Pro key to the configuration that I'm not sure if I simply had my Localstack instance misconfigured or if adding the key caused this to start working.

Reply:
@thrau Do you think mentioning a fleeting note if a config is only for Pro, would be fine?